### ISSUE TITLE
The weights in combined loss function

### ISSUE THREAD

**Issue by Immortals-33** [gxx27/Gated-GPS#1]
Hi, 

Congrats on the work of Gated-GPS! I'm interested in the weight in the combined loss function where $L_{combined} = L_{BCE} + L_{Tversky}$. I've checked the paper and SI to find the $\alpha$ and $\beta$ values within Tversky loss, but did not find the weight between $L_{BCE}$ and $L_{Tversky}$ in the overall loss. Perhaps I miss it?

Additionally I observed in Fig.4 that removing Tversky loss seems to have little effects on the AUPRC values. I'm curious that do you think the positive impact of Tversky loss comes in combination with other mechanisms in the network, e.g. gating module, rather than adding Tversky loss alone without other adjustments of components?

Many thanks in advance!

**Comment by gxx27** [comment]
Hi,

Thanks for your interest in our work!

First, the weight between $L_{CE}$ and $L_{Tversky}$ is 0.3; you can see it from `train.sh`, the `weight` parameter. Also, you can find it in the supplementary files, in the `Table S8`.

Second, yes, I think that the combination of all the components has an overall positive impact on the network, rather than a single module's effect. So, the combined loss really makes sense in some places, but the combination plays an important role in the classification.
